lib/queries/addChallenge.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var fields = ['title', 'description', 'date', 'org_id', 'creator_id', 'active'];
+
+module.exports = function (objectToAdd) {
+  var values = fields.map(function (field) {
+    return '\'' + objectToAdd[field] + '\'';
+  }).join(', ');
+
+  return 'INSERT INTO challenges ('
```

Comment:
The indentation feels a bit strange and makes the query difficult to read I think.